### ISSUE TITLE
[collab] join forces with oxide

### ISSUE THREAD

**Issue by torwag** [rmkit-dev/rmkit#7]
I like remux being rather non-invasive and enabling me to quickly switch between concurrent running apps.
I like [oxide](https://github.com/Eeems/oxide) for bringing me the ``desktop`` like environment 
Why not getting the best of both worlds.
Using oxide as a desktop environment to do settings, starting applications, etc. and use remux as the task-manager to switch between different apps.

Ultimately, if remux could even get to the point of a split view (2 programs side by side) and oxide focuses on the ui experience, we would have a very interesting combination.

**Comment by raisjnn** [comment]
cc @Eeems

**Comment by torwag** [comment]
He got the same feature request ;)

**Comment by Eeems** [comment]
I'm working towards this as well :)
Just got to properly support multiple running applications and to provide some APIs that other applications can use to work with oxide.

**Comment by raisjnn** [comment]
what's the plan? how do we join forces?

**Comment by Eeems** [comment]
Well I'd like to take some time to flesh out my plans for the oxide API and then provide it to you and others for comment. After that I'll start building portions of it for you to start interacting with.

Just a heads up, I'm likely going to be implementing with dbus.

**Comment by torwag** [comment]
As I opened this thread I might want to give my thoughts to it as time passed.
As oxide and remux continue development, I feel that both address different needs.

- oxide is a full `desktop` like starter, icons, menus, status bar. It takes care of settings and contributes 'standard' apps. Thus, in a sense oxide is the KDE of the reMarkable tablet. 

- remux is an `get-out-of-the-way` starter. It appears when needed and disappears instantly to not disturb you with your tasks at hand. It is minimal on design, the i3|awesome|sway (without the tiling) of the reMarkable tablet.

I am pretty happy that the devs share ideas where both programs seem to create an uniform user experience, e.g. trying to establish common standards. 

However, I realized by time, that remux is not a "task manager" of oxide and oxide is not a more visual version of remux. Both address different needs, resp. philosophies and hence should exist in coexistence.
 
So for me, we could close this issue.
